openapi_tester/schema_tester.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         elif "drf_yasg" in settings.INSTALLED_APPS:
             self.loader = DrfYasgSchemaLoader()
         else:
-            raise ImproperlyConfigured("No loader is configured.")
+            raise ImproperlyConfigured("Unable to infer which loader to use.")
```

Comment:
Why is that a better message? it says the same in more words, some of which require better english :).

Reply:
Only because the user cannot "configure" a loader, so it felt hard to act on. 

We can change it to something else, but it's generally nice that the error message hints to the root cause of the error, and I think the current message is a little outdated. 

Do you have any suggestions? We could use `Unable to configure loader` if you prefer that